freeipa/src/main/java/com/sequenceiq/freeipa/entity/SaltSecurityConfig.java

```diff
@@ -83,10 +83,12 @@ public void setSaltBootSignPublicKey(String saltBootSignPublicKey) {
         this.saltBootSignPublicKey = saltBootSignPublicKey;
     }
 
+    @Deprecated
```

Comment:
Can we add some hints why these are deprecated?

Reply:
these should have been removed, I have filed a jira for this to remove them, but I would have to file some sre ticket to validate if I can remove them.